modules/rds-db/main.tf

```diff
@@ -0,0 +1,78 @@
+variable "name_prefix" {
+  type        = string
+  description = "Prefix for all resource names."
+}
+
+variable "subnet_ids" {
+  type        = list(string)
+  description = "Subnets that the database instances live in."
+}
+
+variable "db_instance_type" {
+  type        = string
+  description = "TFE database instance type."
+  default     = "db.m4.large"
+}
+
+variable "db_name" {
+  type        = string
+  description = "Name of TFE database."
+}
+variable "db_username" {
+  type        = string
+  description = "Master username of TFE database."
+}
+variable "db_password" {
+  type        = string
+  description = "Master password of TFE database."
+}
+
+variable "engine" {
+  type        = string
+  description = "Database engine being used."
+}
+
+variable "engine_version" {
+  type        = string
+  description = "Engine version that matches the engine variable."
+}
+
+variable "storage_type" {
+  default     = "gp2"
+  description = "Storage type for the database."
+  type        = string
+}
+
+variable "storage_size" {
+  default     = 50
+  description = "Storage size (GiB) for the database."
+}
+
+variable "sg_ids" {
```

Comment:
In general, the long-form is going to be more obvious and easily discoverable: `security_group_ids` or if singular, `security_group_id`.